### DISCUSSION TITLE
Possibility to convert existing binary masks to labelme format?

### DISCUSSION THREAD

**Discussion by korbinian-hoermann** [wkentaro/labelme#1066] [Ideas / Feature requests]
Hey!

I want to train a custom semantic segmentation model using pixellib. 
Therefore I need annotated lables in a format such as provided by labelme. I do already have some binary masks for the objects I want to segment. 

Is there any way I can convert the information of those binary masks to the annotation style provided by labelme? 

The dataset I want to use is similar to this one: https://www.epfl.ch/labs/cvlab/data/data-em/
![Annotation 2022-01-25 180635](https://user-images.githubusercontent.com/39766418/151024651-02f6a3bc-ca03-4846-bb3e-258563e9a2e0.png)
1st row:   image data
2nd row:  labels as binary mask

Cheers, Korbi :)

**Comment by monkeycc** [comment]
I also want this function

binary masks 
Opencv external contour conversion coordinates
Convert to label JSON
